frontends/blif/blifparse.cc

```diff
@@ -513,6 +515,11 @@ void parse_blif(RTLIL::Design *design, std::istream &f, IdString dff_name, bool
 					sopmode = -1;
 					lastcell = sopcell;
 				}
+				else if (input_sig.size() > lut_input_plane_limit)
+				{
+					err_reason = stringf("names' input plane must have fewer than 13 signals.");
```

Comment:
Please use  `(lut_input_plane_limit + 1)` instead of `13` here

Reply:
err_reason = stringf("names' input plane must have fewer than %d signals.", lut_input_plane_limit + 1);